lib/modules/dosomething/dosomething_rogue/dosomething_rogue.module

```diff
@@ -52,13 +52,14 @@ function _dosomething_rogue_build_http_client() {
  *
  * @param array $values
  *   Values to send to Rogue.
- * @todo - Do we need to pass in user here? Do we need to pass in the item separately?
  */
 function dosomething_rogue_send_reportback_to_rogue($values, $user = NULL) {
   if (!isset($user)) {
     global $user;
   }
 
+  $northstar_user = dosomething_northstar_get_user($user->uid);
```

Comment:
since you don't need the actual Northstar User object here, you could use `dosomething_user_get_field('field_northstar_id', $account)` to just get the ID that's stored locally :world_map: